haas/api.py

```diff
@@ -22,6 +22,7 @@
 from schema import Schema, Optional
 
 from haas import model
+from haas.auth import get_auth_backend
```

Comment:
The main reason I did this is it's consistent with `get_network_allocator()`. I also use a construction `auth_backend = get_auth_backend()` in a number of places, so that should probably change if we were going to change the name.

It also seems like it'd be easy to forget that it's a function without the `get_`; I expect a lot of students forgetting to call it and asking us questions. Shorter is better all else being equal, but you should be careful not to omit important information.

I don't feel strongly. I do have an objection to changing this and not the network allocator one, since I feel the consistency is important.